src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.csproj

```diff
@@ -5,6 +5,7 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
     <NuGetTargetMoniker>.NETStandard,Version=v1.0</NuGetTargetMoniker>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.0;uap10.1</PackageTargetFramework>
```

Comment:
ResourceManager is being built a bit strange today: Contract assembly is being built as 4.0.0.0 despite the implementation being built as 4.0.3.0. I tried to fix it but it causes validation errors. The things we harvest, the things that we prefer to use from live builds, how we define the SupportedFrameworks for target platforms all gets confused when I did that. I tried to manipulate the HarvestStablePackages with the tricks that I know but couldn't have it kneel down. So I'm leaving this weird issue as is, to attentions of @ericstj , @tarekgh , @weshaggard